docs/excel/custom-functions-debugging.md

```diff
@@ -71,8 +69,8 @@ You can use Visual Studio Code to debug custom functions in Office Excel on the
 ### Start debugging
 
 1. In VS code, open your source code script file (functions.js or functions.ts).
-2. In the custom function source code, set a breakpoint on a line of code. You can select a line and choose **Debug -> Toggle Breakpoint**.
-2. In the Excel workbook, enter a formula that uses your custom function.
+2. In the custom function source code, select in the margin next to the code line numbers to set a breakpoint on that line. 
```

Comment:
Same point: "click on the editor margin" instead of "select in the margin".

Reply:
```suggestion
2. In the custom function source code, you can toggle a breakpoint in the margin of the code.
```

Reply:
```suggestion
2. In the custom function source code, set a breakpoint.
```